Testscripts/Linux/utils.sh

```diff
@@ -2434,7 +2434,17 @@ function install_fio () {
 	case "$DISTRO_NAME" in
 		oracle|rhel|centos)
 			install_epel
-			yum -y --nogpgcheck install wget sysstat mdadm blktrace libaio fio
+			yum -y --nogpgcheck install wget sysstat mdadm blktrace libaio fio libaio-devel
+			which fio
```

Comment:
please use these 2 shellcheck recommendations:
1.  SC2230: which is non-standard. Use builtin 'command -v' instead.
2. SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

So lines 2438 and 2439 would go as a simple:
if ! command -v fio; then

Reply:
Thanks Chris .. changed to if ! command -v fio and it works